modules/ocf/manifests/serial_getty.pp

```diff
@@ -1,10 +1,12 @@
 class ocf::serial_getty {
-  service { 'serial-getty@ttyS0':
-    enable   => true,
-    provider => systemd,
-    require  => Package['systemd-sysv'],
-  } ~>
-  exec { 'systemctl start serial-getty@ttyS0':
-    refreshonly => true,
+  if str2bool($::is_virtual) or member(hiera('classes'), 'ocf_kvm') {
```

Comment:
probably not. i wish we had a "role" hiera layer, but such a thing is hard to implement

can you write `member(hiera('classes'), 'ocf_kvm') ` as `tagged('ocf_kvm')` though? i think that is slightly cleaner